### ISSUE TITLE
Deployment issue

### ISSUE THREAD

**Issue by mesmeric** [OrchardCMS/OrchardCore#5607]
Hi all, 

I must be doing something stupid or missing something obvious, so please feel free to mock !

I have an Orchard Core project on my Windows machine and have done a lot of setup and development with it, when running in Debug it works fine, when running in Release it works fine, when running dotnet publish and then running the published files works fine. (Developing in VS2019)

However, I copy the published files to a CentOS machine, configure supervisord and Apache to serve the site and it just shows me the Orchard setup screen.

Database is restored and intact and updated the DB connection string in App_Data/Sites/Default/appSettings.json to point at the restored DB. Creedentials work fine as I am able to login to the DB with them.

I have tried running dotnet publish targeting linux-x64 and portable with the same result.

I am a little confused, what is the most probable cause for the setup screen to be shown? Instead of my developed site? 

I thought I was just missing something on publish, but running the published files on the Windows machine i use for development works fine.

Happy to provide any further info.

Thanks in advance!

**Comment by deanmarcussen** [comment]
Possibly r/w permissions on the `App_Data` folder.

Also check that you have also copied the `tenants.json` file

`appsettings.json` is not camelcased btw

and make sure you have the dataprotection keys folder across (basically everything in `App_Data`)

**Comment by mesmeric** [comment]
Hi Dean

Thank you very much for the response.

App_Data/tenants.json is there

`#cat App_Data/tenants.json`
`
{
  "Default": {
    "State": "Running"
  }
`

App_Data/Sites/Default/appsettings.json is there (not camelcased)

`#cat App_Data/Sites/Default/appsettings.json`

`
{
  "ConnectionString": "Server=localhost;Port=3306;Database=<mydb_name>;Uid=<mydb_user>;Pwd=<mydb_password>",
  "DatabaseProvider": "MySql",
  "TablePrefix": "mpc1"
}
`

App_Data/DataProtection-Keys is also there

I have given the apache user full Write permissions over the App_Data folder and still showing the Setup page :( 
(I would like to know exactly which folders here it needs write access to, in order to lock it down properly though)

Most frustrating!

Many thanks

**Comment by deanmarcussen** [comment]
You might try running through the setup process again on your deployed machine, then assuming it works, resetting the connection strings

**Comment by mesmeric** [comment]
Hi Dean,

I tried running through the setup and switching the connection strings but it did not work. However, It did lead me to the issue.

As it was developed on Windows and then deployed to Linux, Linux is a case sensitive OS so was looking for the camel cased DB table names and not finding them as they were all lower case.

I renamed the tables and it kicked into gear :)

Something for me to watch out for in the future.

Thanks for your help.
